monkey/tests/unit_tests/monkey_island/cc/resources/test_pba_file_upload.py

```diff
@@ -9,7 +9,8 @@
 
 from common import DIContainer
 from monkey_island.cc.repository import IAgentConfigurationRepository, IFileRepository
-from monkey_island.cc.resources.pba_file_upload import LINUX_PBA_TYPE, WINDOWS_PBA_TYPE, FileUpload
+from monkey_island.cc.resources import PBAFileUpload
+from monkey_island.cc.resources.pba_file_upload import LINUX_PBA_TYPE, WINDOWS_PBA_TYPE
```

Comment:
Import these in `__init__.py` as well. The idea is that the internal structure of the package shouldn't be exposed to whatever uses it. I'd expect that includes UTs.